Website/src/rendering/src/components/SpeakerInformation.tsx

```diff
@@ -63,33 +73,41 @@ const SpeakerInformation = (props: SpeakerInformationProps): JSX.Element => (
               )}
             </div>
           </div>
-          <div>
-            <span className="data-label">Position:</span>{' '}
-            <Text field={props.fields.Position}></Text>
-          </div>
-          <div>
-            <span className="data-label">Company:</span> <Text field={props.fields.Company}></Text>
-          </div>
-          <div>
-            <span className="data-label">Country:</span> <Text field={props.fields.Country}></Text>
-          </div>
+          {props.fields.JobTitle && props.fields.JobTitle.value != '' && (
+            <div>
+              <span className="data-label">Job Title:</span>{' '}
+              <Text field={props.fields.JobTitle}></Text>
+            </div>
+          )}
+          {props.fields.Company && props.fields.Company.value != '' && (
+            <div>
+              <span className="data-label">Company:</span>{' '}
+              <Text field={props.fields.Company}></Text>
+            </div>
+          )}
+          {props.fields.Location && props.fields.Location.value != '' && (
+            <div>
+              <span className="data-label">Location:</span>{' '}
+              <Text field={props.fields.Location}></Text>
+            </div>
+          )}
         </div>
         <div className="description-col">
-          {/* TODO: Add speaker type in content hub */}
-          <Text tag="div" className="eyebrow" field={props.fields.Role}></Text>
+          <div>
```

Comment:
You seem to have forgotten to add the eyebrow container to other files that also use the `eyebrow` CSS class like `VendorInformation.tsx`.

Reply:
there was only one div so didnt need to but added it now